internal/pkg/aws/ecs/ecs.go

```diff
@@ -40,8 +79,90 @@ func (s Service) TaskDefinition(taskDefName string) (*TaskDefinition, error) {
 	return &td, nil
 }
 
-// TaskDefinition wraps up ECS TaskDefinition struct.
-type TaskDefinition ecs.TaskDefinition
+// Service calls ECS API and returns the specified service running in the cluster.
+func (e *ECS) Service(clusterName, serviceName string) (*Service, error) {
+	resp, err := e.client.DescribeServices(&ecs.DescribeServicesInput{
+		Cluster:  aws.String(clusterName),
+		Services: aws.StringSlice([]string{serviceName}),
+	})
+	if err != nil {
+		return nil, fmt.Errorf("describe service %s: %w", serviceName, err)
+	}
+	for _, service := range resp.Services {
+		if aws.StringValue(service.ServiceName) == serviceName {
+			svc := Service(*service)
+			return &svc, nil
+		}
+	}
+	return nil, fmt.Errorf("cannot find service %s", serviceName)
+}
+
+// Tasks calls ECS API and returns ECS tasks running in the cluster.
+func (e *ECS) Tasks(clusterName, serviceName string) ([]*Task, error) {
+	var tasks []*Task
+	listTaskResp := &ecs.ListTasksOutput{}
+	for {
+		listTaskResp, err := e.client.ListTasks(&ecs.ListTasksInput{
+			Cluster:     aws.String(clusterName),
+			ServiceName: aws.String(serviceName),
+			NextToken:   listTaskResp.NextToken,
+		})
+		if err != nil {
+			return nil, fmt.Errorf("list running tasks of service %s: %w", serviceName, err)
+		}
+		descTaskResp, err := e.client.DescribeTasks(&ecs.DescribeTasksInput{
+			Cluster: aws.String(clusterName),
+			Tasks:   listTaskResp.TaskArns,
+		})
+		if err != nil {
+			return nil, fmt.Errorf("describe running tasks in cluster %s: %w", clusterName, err)
+		}
+		for _, task := range descTaskResp.Tasks {
+			t := Task(*task)
+			tasks = append(tasks, &t)
+		}
+		if listTaskResp.NextToken == nil {
+			break
+		}
+	}
+	return tasks, nil
+}
+
+// TaskStatus returns the status of the running task.
+func (t *Task) TaskStatus() (*TaskStatus, error) {
```

Comment:
nit:
```suggestion
func (t *Task) Status() (*TaskStatus, error) {
```

Reply:
The reason why I name this one as `TaskStatus` instead of `Status` is to be consistent with `ServiceStatus`. And the reason why I name `ServiceStatus` instead of `Status` is because `ecs.Service` already has a field called `Status`. And golang does not allow both field and method have the same name. So it is a cascade effect...